pkgs/by-name/go/godini/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  godini,
+  lib,
+  nix-update-script,
+  testers,
+}:
+
+buildGoModule rec {
+  pname = "godini";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "bilbilak";
+    repo = "${pname}";
+    rev = "v${version}";
+    hash = "sha256-83OAddIoJzAUXPZKGnAx8XPKrdSmtc1EIJUDmRHTU/U=";
+  };
+
+  vendorHash = "sha256-hocnLCzWN8srQcO3BMNkd2lt0m54Qe7sqAhUxVZlz1k=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/bilbilak/${pname}/config.Version=${version}"
```

Comment:
```suggestion
    "-X github.com/bilbilak/godini/config.Version=${version}"
```

Reply:
Some other packages had used in-string variables as such, so I thought it's alright to do! For the sake of consistency, I replaced this variable with plain text everywhere...